ansible/roles/software/ckan/inventory/tasks/install.yml

```diff
@@ -32,9 +32,10 @@
     pkg: "{{ ckan_pkgs }}"
     virtualenv: "{{ project_source_new_code_path }}"
 
-  # TODO move this upstream somewhere. Maybe ckan's requirements.txt.
-- name: install M2Crypto for pysaml2
-  pip: name=M2Crypto version=0.23.0 virtualenv="{{ project_source_new_code_path }}" state=present
+# TODO move this upstream somewhere. Maybe ckan's requirements.txt.
+# - name: install M2Crypto for pysaml2
+#  pip: name=M2Crypto version=0.23.0 virtualenv="{{ project_source_new_code_path }}" state=present
+#  when: ansible_distribution_release == "trusty"
```

Comment:
This TODO is valid, but have you done it already? Removing this will break existing environments.

Reply:
it is not yet handled upstream will restore the task for now